scripts/generate_grapheme_break_specs.cr

```diff
@@ -1,3 +1,5 @@
+#! /usr/bin/env crystal
```

Comment:
There's no need for space, ditto elsewhere

```suggestion
#!/usr/bin/env crystal
```

Reply:
There are already spaces in the remaining files under `scripts/`